ui/tests/acceptance/enterprise-oidc-namespace-test.js

```diff
@@ -69,23 +65,24 @@ module('Acceptance | Enterprise | oidc auth namespace test', function (hooks) {
     // enable oidc in child namespace with default role
     await authPage.loginNs(this.namespace);
     await this.enableOidc(this.nsOidc, `${this.nsOidc}-role`);
+    // check root namespace for method tab
     await authPage.logout();
-
-    await visit('/vault/auth');
+    await authPage.namespaceInput('');
     assert.dom(SELECTORS.authTab(this.rootOidc)).exists('renders oidc method tab for root');
+    // check child namespace for method tab
     await authPage.namespaceInput(this.namespace);
+    assert.dom(SELECTORS.authTab(this.nsOidc)).exists('renders oidc method tab for child namespace');
+    // clicking on the tab should update with= queryParam
+    await click(`[data-test-auth-method="${this.nsOidc}"] a`);
```

Comment:
This didn't make sense to me how it was passing before. Since we previously logged in with a token the `with` query param is set to `token` and the _Other_ tab is selected so we need to click on the oidc method tab to trigger the request.